plugins/module_utils/network/sonic/config/bgp_ext_communities/bgp_ext_communities.py

```diff
@@ -369,3 +411,84 @@ def validate_type(self, want):
 
                 new_want.append(cfg)
         return new_want
+
+    def get_replaced_overridden_config(self, want, have, cur_state):
+        commands, requests = [], []
+
+        commands_del, requests_del = [], []
+        commands_add, requests_add = [], []
+
+        for conf in want:
+            name = conf['name']
+            in_have = False
+            for have_conf in have:
+                if have_conf['name'] == name:
+                    in_have = True
+                    if have_conf['type'] != conf['type']:
+                        # If both extended community list are of same name but different types
+                        commands_del.append(have_conf)
+                        commands_add.append(conf)
+                    else:
+                        is_change = False
+
+                        if have_conf['permit'] != conf['permit']:
+                            is_change = True
+
+                        if have_conf['match'] != conf['match']:
+                            is_change = True
+
+                        if conf["type"] == "expanded":
+                            members = conf.get('members', {})
+                            if members and conf['members'].get('regex', []):
+                                if have_conf.get('members', {}) and have_conf['members'].get('regex', []):
+                                    if set(have_conf['members']['regex']).symmetric_difference(set(members['regex'])):
+                                        is_change = True
+                            else:
+                                # If there are no members in any extended community list of want, then
+                                # that particular ext community list request to be removed or ignored since
```

Comment:
Please update this part of the comment to match the current implementation.

e.g.
Change "that particular ext community list ..."
to "abort the playbook with an error message explaining why the specified command is not valid."

Reply:
Updated the comment.